sql/src/parser.rs

```diff
@@ -624,6 +637,22 @@ mod tests {
         });
         expect_parse_ok(sql, expected).unwrap();
 
+        // positive case, multiple columns with comment
+        let sql = "CREATE TABLE mytbl(c1 timestamp, c2 double comment 'id', c3 string comment 'name',) ENGINE = XX";
```

Comment:
This case is good, can you also add one in our integration test suit https://github.com/CeresDB/ceresdb/blob/main/tests/cases/local/05_ddl/create_tables.sql?